### ISSUE TITLE
CDN for famo.us university examples is down. 503 Error 

### ISSUE THREAD

**Issue by omasri** [Famous/famous#633]
Getting a 503 for things like http://code.famo.us/famous/0.3/famous-global.min.js
when I trying to go through the famo.us positioning examples.

**Comment by MylesBorins** [comment]
Hey @omasri I'm not able to reproduce this.  We had some weirdness with university today, but it seems to be fixed on our end.  Can you do me a favor and try to reset your cache and post a screen shot if you still have the error?

`shift + cmd (or control) + r`

**Comment by MylesBorins** [comment]
Also could you please provide a link to the page causing issues

**Comment by omasri** [comment]
sorry it was the browser.  The examples works again. going to have some fun exploring this week. closing the issue.

Here's a little example of famo.us "widgets" running in a gazibit presentation layer. 
http://gazibit.com/show/707

I noticed it gives us the same text blurring issue on element transforms.  our layer cleans up the blurr most of the times as you navigate.  We had to fix that issue when we transformed embedded video.
